image-search/Models/AlbumGenerator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AlbumGenerator.swift
+//  image-search
+//
+//  Created by Renato Vieira on 08/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
```

Comment:
Same comment. Is not a "Model". In this case it is better to put in "DataSources"

Reply:
Done